src/CommunityToolkit.Maui.Core/Interfaces/IAvatarView.shared.cs

```diff
@@ -9,6 +9,43 @@ public interface IAvatarView : IBorderView, ILabel, Microsoft.Maui.IImage, IImag
 	/// <summary>Gets a value indicating the avatar border width.</summary>
 	double BorderWidth { get; }
 
+	/// <summary>Gets or sets a value of the control image cache validity.</summary>
+	TimeSpan CacheValidity { get; }
```

Comment:
Do we need it? The ImageSource already has this out of the box as you can see [here](https://docs.microsoft.com/en-us/dotnet/maui/user-interface/controls/image#image-caching)

Reply:
Removed unnecessary properties, added examples, added GravatarEmail property instead.